src/Compilers/VisualBasic/Portable/Errors/Errors.vb

```diff
@@ -72,6 +72,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         ERR_MutuallyExclusiveOptions = 2046
         ERR_BadSwitchValue = 2047
 
+        ERR_InvalidVersionFormatDeterministic = 8357
```

Comment:
@333fred do we normally try and match error codes like this? Semes like a fairly high jump

Reply:
No, we make no attempts to match error codes between C# and VB.

Reply:
Thanks! That is new information to me and unexpected. Would I assign 2048? or jump to the bottom and use 37331 ?

Reply:
37330 please, and increment ERR_NextAvailable to 37331.